ddtrace/tracer/textmap_test.go

```diff
@@ -706,8 +713,9 @@ func TestEnvVars(t *testing.T) {
 					defer tracer.Stop()
 					root := tracer.StartSpan("web.request").(*span)
 					ctx, ok := root.Context().(*spanContext)
-					ctx.traceID = test.in[0]
-					ctx.spanID = test.in[1]
+					ctx.traceID128 = fmt.Sprintf("%016x", test.in[0]) // add when 128-bit trace id support is enabled
```

Comment:
should this be commented out like the above assertion that has this comment?

Reply:
It's not hurting anything being in the test right now. It'll just set it to zero until it's implemented 🤷